airflow/api_connexion/exceptions.py

```diff
@@ -0,0 +1,66 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+def response_404(
```

Comment:
I think, we should use exception to handle errors. 
I have prepared an example that shows how to do it.
https://termbin.com/legq
To add this commit to your brach, run the following command:
```
curl https://termbin.com/legq | git am
```

Reply:
Thank you, this exactly what I was stuck on.

> [](https://termbin.com/legq)
> To add this commit to your brach, run the following command:
> 
> ```
> curl https://termbin.com/legq | git am
> ```

Thank you this is exactly what I was stuck on.

Reply:
Fixed `00da53d `

Reply:
Added other http exception classes `cd126b7`